development-docs/DEVELOPING.md

```diff
@@ -13,38 +13,27 @@ You can run the application in dev mode that enables live coding using:
 mvn compile quarkus:dev
 ```
 
-### Creating a native executable using Maven
+### Building the Java code with Maven
 
-If you have [GraalVM](https://www.graalvm.org/) installed locally, you can create a native executable using: 
+You can build the project using Maven.
+Once you have Java and Maven installed, you can run to build the project  
 ```shell script
-mvn package -Pnative
+mvn package
 ```
 
-Or you can run the native executable build for Linux in a container using: 
-```shell script
-mvn package -Pnative -Dquarkus.native.container-build=true
-```
-
-This is useful especially when running on other operating systems such as macOS or when you don't have GraalVM installed.
-
-You can then execute your native executable with: `./target/strimzi-drain-cleaner-1.0.0-SNAPSHOT-runner`.
-
 ### Creating a native executable using Make
```

Comment:
If we're removing native image support I guess we need to remove this heading too?